### ISSUE TITLE
Popovers and Tooltips - problem at the 2nd and subsequent clicks.

### ISSUE THREAD

**Issue by afuno** [twbs/bootstrap#19790]
Example: https://jsfiddle.net/mka8LLyr/

Click the first time. It's okay.
Then defocusing it.

Again, push and catch the error: [see](https://habrastorage.org/files/cb8/6b3/97a/cb86b397a040400280c1bd66cda3c7f8.png)

This applies to Popovers. A similar problem and Tooltips.

**Comment by twbs-lmvtfy** [comment]
Hi @afuno!

You appear to have posted a live example (https://fiddle.jshell.net/mka8LLyr/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 59, column 3 thru column 17: Stray doctype.
* line 60, column 1 thru column 16: Stray start tag ```` html ````.
* line 61, column 3 thru column 8: Stray start tag ```` head ````.
* line 62, column 5 thru column 26: Attribute ```` charset ```` not allowed on element ```` meta ```` at this point.
* line 62, column 5 thru column 26: Element ```` meta ```` is missing one or more of the following attributes: ```` content ````, ```` itemprop ````, ```` property ````.
* line 63, column 5 thru column 90: Attribute ```` name ```` not allowed on element ```` meta ```` at this point.
* line 63, column 5 thru column 90: Element ```` meta ```` is missing one or more of the following attributes: ```` itemprop ````, ```` property ````.
* line 64, column 5 thru column 57: Attribute ```` http-equiv ```` not allowed on element ```` meta ```` at this point.
* line 64, column 5 thru column 57: Element ```` meta ```` is missing one or more of the following attributes: ```` itemprop ````, ```` property ````.
* line 65, column 5 thru column 220: Attribute ```` integrity ```` not allowed on element ```` link ```` at this point.
* line 65, column 5 thru column 220: Element ```` link ```` is missing required attribute ```` property ````.
* line 69, column 3 thru column 9: Stray end tag ```` head ````.
* line 70, column 3 thru column 8: Start tag ```` body ```` seen but an element of the same type was already open.
* line 90, column 1 thru column 7: Stray end tag ```` body ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by wolfy1339** [comment]
Fixed example: https://jsfiddle.net/mka8LLyr/2/

**Comment by twbs-lmvtfy** [comment]
Hi @wolfy1339!

You appear to have posted a live example (https://fiddle.jshell.net/mka8LLyr/2/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 75, column 3 thru column 23: A ```` charset ```` attribute on a ```` meta ```` element found after the first 512 bytes.
* line 75, column 3 thru column 24: Attribute ```` charset ```` not allowed on element ```` meta ```` at this point.
* line 75, column 3 thru column 24: Element ```` meta ```` is missing one or more of the following attributes: ```` content ````, ```` itemprop ````, ```` property ````.
* line 76, column 1 thru column 53: Attribute ```` http-equiv ```` not allowed on element ```` meta ```` at this point.
* line 76, column 1 thru column 53: Element ```` meta ```` is missing one or more of the following attributes: ```` itemprop ````, ```` property ````.
* line 77, column 1 thru column 68: Attribute ```` name ```` not allowed on element ```` meta ```` at this point.
* line 77, column 1 thru column 68: Element ```` meta ```` is missing one or more of the following attributes: ```` itemprop ````, ```` property ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by gregblass** [comment]
See my code that I just posted here for my workaround:

https://github.com/twbs/bootstrap/issues/16028

**Comment by afuno** [comment]
@gregblass Something I did not understand a little bit, what is the difference of my piece of code from your.

**Comment by RyanZim** [comment]
@afuno, I cannot reproduce in Firefox, Chrome, or Safari on Windows when all the dependencies are loaded correctly. What browser and OS are you using?

**Comment by afuno** [comment]
@RyanZim I have been using the latest version of OS X and Safari. @gregblass example for me works is true. My example I have is not working. What is the difference between them?

**Comment by cvrebert** [comment]
Using GitHub URLs such as https://raw.githubusercontent.com/HubSpot/tether/master/dist/js/tether.js in `<script>` tags, as the 1st fiddle attempted, doesn't work. See https://github.com/blog/1482-heads-up-nosniff-header-support-coming-to-chrome-and-firefox . GitHub isn't a CDN.
Anyway, I can't reproduce the alleged bug with https://fiddle.jshell.net/mka8LLyr/3/ in Safari 9.1 on OS X El Capitan.

**Comment by afuno** [comment]
@cvrebert I use Bootstrap 4 alpha 3 as a gem for Rails.

**Comment by cvrebert** [comment]
@afuno Then I encourage you to seek debugging help from the community (see https://github.com/twbs/bootstrap/blob/v4-dev/README.md#community ), and if your problem turns out to be reproducible by others, then feel free to open a new issue with a new testcase.
